core/src/Revolution/Sources/modMediaSource.php

```diff
@@ -1087,10 +1101,10 @@ public function uploadObjectsToContainer($container, array $objects = [])
 
         foreach ($objects as $file) {
             $this->xpdo->invokeEvent('OnFileManagerBeforeUpload', [
-                'files' => &$objects,
```

Comment:
I think this should be reverted to allow plugins to change the values (in particular $file) by reference, unless you had a good reason to remove it that I'm failing to see?

Reply:
Right, reverting that now.